apps/modernization-ui/src/pages/patient/profile/TableActions.tsx

```diff
@@ -0,0 +1,30 @@
+import { Button, Icon } from '@trussworks/react-uswds';
+import { Actions } from 'components/Table/Actions';
+import { Patient } from './Patient';
+
+type ActionType = 'add' | 'update' | 'delete' | 'details';
+
+type ActionProps = {
+    patient: Patient | undefined;
+    index: number;
+    handleAction: (type: ActionType) => void;
+    setActionIndex: (action: number | null) => void;
+    actionIndex: number | null;
+};
+
+export const TableActions = ({ patient, index, handleAction, setActionIndex, actionIndex }: ActionProps) => {
```

Comment:
TableActions name here is too generic while this component is dealing with a patient data. Not a blocker though.